include/cfdcore/cfdcore_json_mapping_base.h

```diff
@@ -135,11 +135,20 @@ inline void ConvertFromUniValue(
 template <typename T>
 inline void ConvertFromUniValue(
     T& value, const UniValue& json_value) {  // NOLINT
-  if (json_value.isNum()) {
+  UniValue json_value_copy = json_value;
+  if (json_value_copy.isStr()) {
+    std::string str = json_value.get_str();
+    bool is_digits_only = std::all_of(str.begin(), str.end(), ::isdigit);
+    if (is_digits_only) {
```

Comment:
I think it's better to add a string length check.
maximum length of uint64 and int64 is 20 charactor.
```suggestion
    if (is_digits_only && (str.length() <= 20)) {  // max of uint64 and int64
```

Reply:
Updated. My thinking was that an exception would be thrown if the string is invalid. Actually, I feel that instead of silently setting `value = 0` line 175 it might be better to throw (it took me some time to realize why the values in the C++ layer where at 0).

Reply:
You're right. :pray:
After verifying the implementation of Univalue, I want to add a check.